theme_vehicle/views/customizations.xml

```diff
@@ -1,171 +1,352 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <!-- ======== BANNER ======== -->
+    <template id="s_banner" inherit_id="website.s_banner">
```

Comment:
For new files, always no indentation of the first level please. In this case, it was already good, just had to stay with no indentation.